### PR TITLE
Adds a flight data recorder (tldr; new GPS for ruins)

### PR DESCRIPTION
**Pull request by TMTIME**
[Changelogs]: 

:cl: TMTIME
add: A new type of GPS, the flight data recorder
add: The FDR can be sold at stations
add: Placed FDRs in ruins that would likely have one
imageadd: Quick sprite for the FDR
/:cl:

[why]: It's super uncommon for players to stumble upon ruins ~~ever since the fix for ruins spawning under the ship was merged~~ so this PR aims to make it easier to find them and reward crews who investigate them.

The flight data recorder.

By default, the FDR has a temp gpstag but can be edited either by mappers or admins who are making an event.

The FDR can currently be sold for 1500 at a station, and has the Salvage keyword applied.

The following ruins have had FDRs added, as they were ships that could realistically have them:

- bigderelict1
- caravanambush
- crashedclownship
- crashedship
- derelict1
- derelict4
- intactemptyship
- onehalf
- TheDerelict

~~also im sorry if more clowns get honk mechs because of this~~
~~i also made one ruin slightly more dangerous~~ 

Edit: To clarify the change to the init of GPSs, without that the FDRs would be called 'global positioning system (Emergency transponder - X)'. The change doesn't affect the other GPSs

### CODE REVIEW

File: _maps/RandomRuins/SpaceRuins/TheDerelict.dmm [hunk 1]
```diff
@@ -5057,6 +5057,14 @@
 /obj/structure/cable,
 /turf/open/floor/plasteel/airless,
 /area/solar/derelict_aft)
+"oE" = (
+/obj/item/device/gps/flightdatarecorder{
+	gpstag = "Emergency Transponder - Kosmicheskaya Stantsiya 13"
```
Comment (monster860) [inline _maps/RandomRuins/SpaceRuins/TheDerelict.dmm]: I was going to yell at you for not using actual cyrillic characters before realizing that BYOND sucks and you cant.

File: code/game/objects/items/devices/gps.dm [hunk 2]
```diff
@@ -160,6 +160,13 @@ GLOBAL_LIST_EMPTY(GPS_list)
 	desc = "A mining cyborg internal positioning system. Used as a recovery beacon for damaged cyborg assets, or a collaboration tool for mining teams."
 	flags = NODROP
 
+/obj/item/device/gps/flightdatarecorder
+	name = "flight data recorder"
+	icon_state = "gps-fdr"
+	w_class = WEIGHT_CLASS_BULKY
+	gpstag = "Emergency Transponder - X"
```
Comment (monster860) [inline code/game/objects/items/devices/gps.dm]: Aren't gps tags usually 4 letters?

Comment (TMTIME) [reply code/game/objects/items/devices/gps.dm]: Nope, there is a character limit when a player sets it (20), but can display huge lines of text

dreamseeker_2017-11-06_17-07-38

Comment (monster860) [reply code/game/objects/items/devices/gps.dm]: Yeah but you should probably stick with the usual format of 4-letter tags

Comment (TMTIME) [reply code/game/objects/items/devices/gps.dm]: I could get it down to 'BBET' but that's very vague. 'FDR-Emergency' is shorter and clearer as to what it is, if you are alright with that?

Comment (monster860) [reply code/game/objects/items/devices/gps.dm]: Yeah, sure

**Comment by TMTIME** [comment]
part two soon, redoing the maps

**Comment by TMTIME** [comment]
All done? I thought I missed one as the changelog showed one on a map that still had the old gpstag but isn't actually placed

**Comment by monster860** [comment]
By the way, your username goes after :cl: not a description of the PR

**Comment by TMTIME** [comment]
Done

**Comment by monster860** [comment]
@Qustinnus any issues?

**Pull request merged**